packages/documentation/src/content/docs/admin/liquidity/asset-liquidity.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Asset liquidity
+tableOfContents:
+  maxHeadingLevel: 4
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components'
+import { CodeBlock, LinkOut } from '@interledger/docs-design-system'
+import IdempotencyNote from '/src/partials/liquidity-idempotency.mdx'
+
+Asset liquidity is the amount of value, denominated in a given asset, that Rafiki has available to handle cross-currency (foreign exchange) transactions between you and your peer. An asset's liquidity increases when Interledger packets are received and decreases when packets are sent/forwarded.
```

Comment:
> An asset’s liquidity increases when Interledger packets are received and decreases when packets are sent/forwarded.

Technically, funds will always move between two asset liquidity accounts, so one asset liquidity will go up, the other down. We can say something like, "whenever an outgoing payment/incoming payment is in a different asset that the peering relationship, the liquidity of asset accounts will change, depending on the FX direction"

Reply:
Updated this page and a similar piece within the accounting concepts page.